docs/content/en/configuration/notifications/smtp.md

```diff
@@ -132,6 +133,13 @@ to leave this as is, but you can customize it if you have issues or you desire t
 For security reasons the default settings for Authelia require the SMTP connection is encrypted by TLS. See [security]
 for more information. This option disables this measure (not recommended).
 
+### disable_starttls
+
+{{< confkey type="boolean" default="false" required="no" >}}
+
+For security reasons the default settings for Authelia require the SMTP connection do use STARTTLS when is available. See [security]
+for more information. This option disables this measure (not recommended).
```

Comment:
```suggestion
Some SMTP servers ignore SMTP specifications and claim to support STARTTLS when they in fact do not.
For security reasons Authelia refuses to send messages to these servers.
This option disables this measure and is enabled  *__AT YOUR OWN RISK__*. It's *__strongly recommended__* 
that instead of enabling this option you either fix the issue with the SMTP server's configuration or 
have the administrators of the server fix it. If the issue can't be fixed by configuration we recommend 
lodging an issue with the authors of the SMTP server.
See [security] for more information.
```